sensor.py

```diff
@@ -6,10 +6,13 @@
 from custom_components.smartthinq import (
     CONF_LANGUAGE, KEY_SMARTTHINQ_DEVICES, LGDevice)
 import homeassistant.helpers.config_validation as cv
-from homeassistant.const import CONF_REGION, CONF_TOKEN
+from homeassistant.const import CONF_REGION, CONF_TOKEN, DEVICE_CLASS_POWER, POWER_WATT
 
-import wideq
-from wideq import dishwasher
+from . import wideq
+from .wideq import dishwasher
+from .wideq import ac
```

Comment:
The `.` and `.wideq` don't quite look right—wouldn't that require us to copy the wideq code here?